eg/eg_exec.py

```diff
@@ -1,10 +1,15 @@
 #!/usr/bin/python
 import argparse
-import eg_config
-import eg_util
 import pydoc
 import sys
 
+try:
+    import eg_config
+    import eg_util
+except ImportError:
+    from eg import eg_config
+    from eg import eg_util
+
```

Comment:
From what I understand, the relative dot imports are really just a stylistic choice. The main advantage is that if you change the package name, you don't have to fix a bunch of imports. If you don't plan on changing the name or wanting to leave that flexibility, there's really no reason to use the dot syntax. I'll leave the decision up to you as the package author.